diff --git a/dist/index.mjs b/dist/index.mjs
index 476ac0fbba81d7970b0a185e7f0eb8f3cbca10f6..230ffd46ea0234d76bb6bb954b4b1126893e5be3 100644
--- a/dist/index.mjs
+++ b/dist/index.mjs
@@ -112,7 +112,7 @@ var correctArgsPosition = {
               correctUsage
             },
             fix(fixer) {
-              const sourceCode = context.getSourceCode();
+              const sourceCode = context.sourceCode;
               const firstArgText = sourceCode.getText(args[0]);
               const secondArgText = sourceCode.getText(args[1]);
               const fixes = [
